migrations/1617983833835-add-default-advanced-trigger-settings.js

```diff
@@ -0,0 +1,21 @@
+const { saveConfig } = require('../lib/new-settings-loader')
+
+exports.up = function (next) {
+  const triggersDefault = {
+    triggersConfig_expirationTime: 'Forever',
+    triggersConfig_automation: 'Automatic'
+  }
+
+  return saveConfig(triggersDefault)
+    .then(() => next())
+    .catch(err => {
+      if (err.message === 'lamassu-server is not configured') {
```

Comment:
This should never happen, it's stuff from the old settings-loader. Even if it's possible to fail on the new-settings-loader we want to break the migration here in case of error.

Reply:
So this if shouldn't exist? I've doing this for every migration. The only thing missing here is the return next(err) at the end:
    .catch(err => {
      if (err.message === 'lamassu-server is not configured') {
        return next()
      }
      console.log(err.message)
      return next(err)
    })

Reply:
What other migration has this? The `lamassu server is not configured` part is related to the 7.4 config schema so it should only appear on the 7.4 -> 7.5 config migration.

Reply:
https://github.com/lamassu/lamassu-server/pull/704/files This one for instance. I think it needs fixing too.